examples/angular/src/pages/ui/components/authenticator/sign-up-with-phone/sign-up-with-phone.component.html

```diff
@@ -1 +1,10 @@
-<amplify-authenticator></amplify-authenticator>
+<amplify-authenticator>
+  <ng-template
+    amplifyOverride="authenticated"
+    let-user="user"
+    let-signOut="signOut"
+  >
+    <h2>Welcome, {{ user.username }}!</h2>
+    <button (click)="signOut()">Sign Out</button>
+  </ng-template>
```

Comment:
This looks good. I like the idea of exposing signOut and user. Maybe in our discussion today we can chat on what other things we need to expose through slots